### PR TITLE
auto scroll set to false when reaching camera boundaries

### PR DESCRIPTION
**Pull request by daniel-mf**
`autoScrollX` and `autoScrollY` were `true` when there was no movement at all

### CODE REVIEW

**Comment by jdnichollsc** [comment]
@daniel-mf Ohh excellent man! Can you modify the version of the plugin (.js file and package.json) to plublish to npm and add you as a contributor from the README?
Thanks for your contribution! 👍

**Comment by daniel-mf** [comment]
@jdnichollsc Sure! I'll be doing that by the end of today.

**Comment by daniel-mf** [comment]
@jdnichollsc done 👍

**Comment by jdnichollsc** [comment]
@daniel-mf only to confirm, why `1.0.8` instead of `1.0.7`? Thanks man!

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "phaser-kinetic-scrolling-plugin",
-  "version": "1.0.3",
+  "version": "1.0.5",
```
Comment (jdnichollsc) [inline bower.json]: @daniel-mf I think we need tu use the same version, what do you think?

Comment (daniel-mf) [reply bower.json]: I usually upgrade version number based on the changes, what's why 0.0.2 for 2 new methods...
I'm getting bower up to 1.0.8 too.
Is that ok?

**Comment by daniel-mf** [comment]
@jdnichollsc I usually upgrade version number based on the changes, that's why 0.0.2 for 2 new methods.
Is that ok?

File: bower.json (hunk 1 above)
Comment (daniel-mf) [reply bower.json]: Done.

**Comment by jdnichollsc** [comment]
@daniel-mf  Sounds perfect, thanks for your help! 💯

**Pull request merged**